packages/cf.js/test/app-factory.spec.ts

```diff
@@ -27,12 +27,12 @@ describe("CF.js AppFactory", () => {
   });
 
   describe("proposeInstall()", () => {
-    it("should correctly propose valid app installs", async () => {
+    it("can propose valid app installs", async () => {
       expect.assertions(4);
       nodeProvider.onMethodRequest(Node.MethodName.PROPOSE_INSTALL, request => {
         expect(request.type).toBe(Node.MethodName.PROPOSE_INSTALL);
         const params = request.params as Node.ProposeInstallParams;
-        expect(params.initialState).toBe("1559");
+        expect(params.initialState).toBe("4000");
```

Comment:
I would refer to constant variables in the test file versus literal numbers in the `it` block. e.g., I have no clue what "4000" represents when reviewing this PR